### ISSUE TITLE
Report stats based on what is in a graph

### ISSUE THREAD

**Issue by valentinedwv** [earthcube/earthcube_utilities#139]
basically, a community/tenant would show  sources in it's graph, and the Geocodes would show all loaded sources. 

   *  a manual (weekly task) would build a quick summary of the sources loaded into the a community graph , and in put it in an s3 bucket. 
   *  The (JSON?) file pushed to the s3 repo may have more than just the graph summary from the spaql query,
            * it will have the descriptions, logos, and other information pulled from the 'sources' googlesheet.
The report file should allow for a community name. default graph namespace (eg earthcube, deepoceans). 
`report_{community}_stats.json`
 
Sparql queries for the about page need to be run over the graph store.

Ideally, we would run the all_graph example, but these kill the graph
[datasets](https://github.com/earthcube/earthcube_utilities/blob/dev/earthcube_utilities/src/ec/graph/sparql_files/all_repo_count_datasets.sparql)
[all graphs](https://github.com/earthcube/earthcube_utilities/blob/dev/earthcube_utilities/src/ec/graph/sparql_files/all_repo_count_graphs.sparql)

Since these can be graph killers, so we may want to extract all graph uris using sparql below, and grabbing say 10000 at a time, and process using a dataframe. (this will not work for all triplestores, it will fail in graphdb, but we are not using graphdb)

```
SELECT   ?g
WHERE     {
  GRAPH ?g {
      
   }
} order by ?g
limit 10000

```

with offset
```
SELECT   ?g
WHERE     {
  GRAPH ?g {
      
   }
} order by ?g
limit 10000

offset 10000
```

So, this will probably need to use the code, because of the limit and offset and merging into a single dataframe
or maybe not... parameters object 
{ "limit": 10000, "offset": the offset }

https://github.com/earthcube/earthcube_utilities/blob/e940603b75a330e09406c27595087f1db3b9ba78/earthcube_utilities/src/ec/graph/sparql_query.py#L28-L34

add this query to the sparql files. all_graphs_offset_limit.sparql
```
SELECT   ?g
WHERE     {
  GRAPH ?g {
      
   }
} order by ?g
limit ${limit}

offset ${offset}
```

**Comment by ylyangtw** [comment]
The current code works. Need to improve performance.

**Comment by ylyangtw** [comment]
Maybe using data from graph reports makes sense? That way we don't need to calculate them again.

**Comment by valentinedwv** [comment]
That does make some sense, since it basically does this.

But we still want to know what is in the graph being used by a user interface. There could accidentally be an additional source loaded into the graph. Only way to know it to get them all.

**Comment by ylyangtw** [comment]
closed this issue as it's been addressed in #141
